.github/workflows/pipeline.yml

```diff
@@ -72,9 +72,7 @@ jobs:
           useLockFile: false
 
       - name: Install ESLint v${{ matrix.eslint }}
-        # force installation for now until we get ESLint and all plugins updated
-        # in dev dependencies
-        run: npm install --no-save --force eslint@${{ matrix.eslint }}
```

Comment:
Sorry, I just saw that [`eslint-config-kentcdodds` is compatible only with ESLint v8](https://github.com/kentcdodds/eslint-config-kentcdodds/blob/f28506ef913486c19fd4c69d8e1eac80cc5e4ee4/package.json#L63), so we can't remove the `--force` flag here.

Can you put it back? The comment is not up to date tho, it should be something like "force installation to skip dev dependencies not compatible with ESLint v7"